app/globals.css

```diff
@@ -210,8 +215,15 @@ button {
   height: 100%;
   position: fixed;
   z-index: -1;
+  background-size: cover;
 }
 
 .fullBGpicture > main {
   overflow: auto;
-}
\ No newline at end of file
+}
+
+.title-icons {
```

Comment:
Usually the `globals.css` file is used for css classes which will be used over and over again in the entire application. 
If you are going to use the class only on 1 page it doesn't belong here.